javascript-data-model/order-history.js

```diff
@@ -0,0 +1,63 @@
+var orderHistory = [{
+  deliveryDate: 'August 8, 2020',
+  items: [{
+    name: 'JavaScript for impatient Programmers',
+    type: 'book',
+    orderPlaced: 'August 4th, 2020',
+    totalCost: '$34.00',
+    orderNumber: 114 - 3941689 - 8772232,
+    author: 'Dr. Axel Rauschmauer'
+  }]
+
+},
+
+{
+  deliveryDate: 'July 20th',
+  items: [{
+    name: 'The Timeless way of building',
+    type: 'book',
+    orderPlaced: 'July 19th 4th, 2020',
+    totalCost: 41.53,
+    orderNumber: 113 - 9984268 - 1280257,
+    author: 'Christopher Alexander'
+
+  }]
+
+},
+
+{
+  deliveryDate: 'July 7th',
+  items: [{
+    name: 'GameCubeAdaptor',
+    type: 'adaptor',
+    orderPlaced: 'July 4th, 2020',
+    totalCost: 17.22,
+    orderNumber: 114 - 3941689 - 8772232
+  }]
+},
+
+{
+  deliveryDate: 'July 5th',
+  items: [{
+    name: 'GameCube',
+    type: 'controller',
+    orderPlaced: 'July 3rd',
+    edition: 'Nintendo Switch',
+    totalCost: 94.95,
+    orderNumber: 113 - 2883177 - 264826
+  }]
+},
+
+{
+  deliveryDate: 'July 5th',
+  items: [{
+    name: 'The Art of Sql',
+    type: 'book',
+    author: 'Stephane Faroult',
+    totalCost: 33.99,
+    orderNumber: 113 - 2883177 - 264826,
+    orderPlaced: 'July 3rd'
+
+  }]
+}];
```

Comment:
This looks like two separate orders right now, you want one order object for each order which will hold an array of all items in that order.